### PR TITLE
[Ethics] Starting to add content

### PR DESCRIPTION
**Pull request by carriewright11**
#### What changes are being implemented in this Pull Request?

Starting to add the ethics chapter.

#### What was your approach?

Took some text from @ehumph and then added to it and rearranged things. Structuring based on major concerns as a general list at first and then describing each in more detail, as well as strategies to mitigate or avoid the issues. Need to make sure we add lots of disclaimers that there may be more concerns that have yet to be revealed. I also want to add info from papers like this: https://www.technologyreview.com/2023/04/18/1071727/generative-ai-risks-concentrating-big-techs-power-heres-how-to-stop-it/

I also want to add examples of typical issues - like examples of how code and data already are biased.

### Tell potential reviewers what kind of feedback you are soliciting.

- Anything missing?
- Anything you would change?

### CODE REVIEW

**Comment by ehumph** [comment]
@carriewright11 I noticed there was a conflict between the bookdown in this PR and the one now on main, so I fixed the conflict. It was calming.

**Comment by carriewright11** [comment]
> @carriewright11 I noticed there was a conflict between the bookdown in this PR and the one now on main, so I fixed the conflict. It was calming.

somehow I still saw a conflict

**Comment by ehumph** [comment]
> > @carriewright11 I noticed there was a conflict between the bookdown in this PR and the one now on main, so I fixed the conflict. It was calming.
> 
> somehow I still saw a conflict

That is because github is a delicate flower.

**Comment by carriewright11** [comment]
I didn't yet add info about this though: https://www.technologyreview.com/2023/04/18/1071727/generative-ai-risks-concentrating-big-techs-power-heres-how-to-stop-it/

**Review by ehumph — APPROVED** [review]
LGTM! I made some formatting changes, but I seem to have committed them to the branch instead of leaving them as comments for you to choose whether you want to commit them. Not really sure how I did that, but it's late and it's very possible I am not thinking the most clearly atm.

You made some excellent points, especially the one about human programmers recognizing their own strengths!

**Comment by ehumph** [comment]
@carriewright11 take a look at the changes and if you don't like 'em, feel free to change it back.

ALSO! I think the ethics chapter should come right after the introduction instead of at the end.

**Comment by carriewright11** [comment]
> @carriewright11 take a look at the changes and if you don't like 'em, feel free to change it back.
> 
> ALSO! I think the ethics chapter should come right after the introduction instead of at the end.
> 
> What do you think about calling the chapter "Ethics of Using AI"?

all of that sounds good to me!

File: 07-ethics.Rmd [hunk 1]
```diff
@@ -0,0 +1,124 @@
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Ethics of AI
```
Comment (carriewright11) [inline 07-ethics.Rmd:6]:
```suggestion
# Ethics of Using AI
```

File: 02-ethics.Rmd [hunk 2]
```diff
@@ -3,29 +3,50 @@
...
+1) **Copyright Violations** - The code used for the generative AI model may used code that has copyright laws that require attribution or do not allow reuse and it may not be clear where the code came from.
+1) **Harmful Code** - Currently it is not clear how well generative AI models restrict the creation of code that will be used for goals that harm others.
 
 
-Note that this is an incomplete list; additional ethical concerns will become apparent as we continue to use these new technologies.
+Note that this is an incomplete list; additional ethical concerns will become apparent as we continue to use these new technologies. We highly suggest that users of these tools be **transparent** about the use of these tools, so that as new ethical issues emerge, we will be better prepared to understand the implications for specific coding projects.
+
+:::{.ethics}
+Be transparent about what AI tools you are using. Not only does it help others to better understand how you created your code and what possible code written by humans the tool actually used in generating the code, but it also may help with future issues related to the use of these tools.
+:::
```
Comment (ehumph) [inline 02-ethics.Rmd:33]: I love the placement of this box, but the wording is a little confusing. Maybe "Be transparent about what AI tools you are using. Not only does it help others to better understand how you created your code and the possible sources the AI is using when it proposes code, but it also may help with future issues related to the use of these tools."

File: 02-ethics.Rmd [hunk 3]
```diff
@@ -3,29 +3,50 @@
...
+
+Computer science is a field that has historically lacked diversity and new learners can often feel intimidated. There is the potential that new learners may feel even more discouraged as they learn to write code when witnessing AI tools write code. It is critical that we support diverse new learners of computer science, as we will continue to need human involvement in the development and use of AI tools.
 
+### Tips for supporting human code
+
+* Follow best teaching practices for inclusion of diverse learners.
+* Encourage current and future programmers to continue to innovate and write code. 
+* Avoid thinking that code written by computers must be better than code written by humans, as this simply is not true. 
+* Recall that humans first wrote they code that the AI tools use to write code.
+* Be transparent about when you do or do not use AI tools, give credit to the humans who write code where possible.
```
Comment (ehumph) [inline 02-ethics.Rmd:50]: I'm glad you added this section and made it first - it really is the critical message we need to hammer home.

**Review by ehumph — APPROVED** [review]
LGTM!

File: 02-ethics.Rmd (hunk 2 above)
Comment (carriewright11) [reply 02-ethics.Rmd:33]: I updated the wording.

**Pull request merged**